uaclient/config.py

```diff
@@ -214,17 +214,18 @@ def _unattached_status(self) -> "Dict[str, Any]":
                 available = status.UserFacingAvailability.AVAILABLE.value
             else:
                 available = status.UserFacingAvailability.UNAVAILABLE.value
-            ent_cls = ENTITLEMENT_CLASS_BY_NAME.get(resource["name"])
+            alias_name = resource["name"]
```

Comment:
What's "alias" in this case?

Reply:
oops vestigial from the alias portion of this branch which was going to alias an type: 'esm' from the backend as 'esm-infra' in the UA client if the backend had not switched over to the new key name. Backend has already landed that component and surfaced esm-infra it in staging so we don't need it. Dropping this remaining alias-related declaration